models/apartment.rb

```diff
@@ -0,0 +1,20 @@
+class Apartment < ActiveRecord::Base
+  has_many :tenants
+  def initialize(address, monthly_rent, sqft, num_beds, num_baths)
```

Comment:
We actually don't need this method since active record sets all our our attributes as attr_accessors by default.